repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -721,18 +722,50 @@ export const getPassengerTypesByNocCode = async <T extends keyof SavedPassengerT
 
     try {
         const queryInput = `
-            SELECT contents, name
+            SELECT name, contents
             FROM passengerType
             WHERE nocCode = ?
             AND isGroup = ?
         `;
 
-        const queryResults = await executeQuery<{ contents: string; name: string }[]>(queryInput, [
+        const queryResults = await executeQuery<{ name: string; contents: string }[]>(queryInput, [
```

Comment:
What's the reason for this and the change above? Is the order important?

Reply:
![image](https://user-images.githubusercontent.com/42877376/127503924-32ce9d9c-8dd0-457a-9a27-f15e4e32e387.png)

No reason other than my slight OCD. I wanted the order to match the database. I'm happy to revert if you like?